app/views/account/companies/pdf.html.slim

```diff
@@ -0,0 +1,56 @@
+.content-wrapper
+  .content-body.container
+    .row
+      .card.container
+        .card-header.row
+          h4.card-title Overview
+        .card-content.collapse.show
+          .card-body.container
+            .row
+              .col-3
+                img.img-fluid src="https://crunchbase-production-res.cloudinary.com/image/upload/c_lpad,h_120,w_120,f_auto,q_auto:eco/v1459804290/mkxozts4fsvkj73azuls.png"
+              .col-9
+                h4= company.name
+                a href="http://#{company.domain}"= company.domain
```

Comment:
why don't you just wrap html attributes into a brackets to make it easier to work with
e.g.:
```
a[href="some.where.com" title="Click Me!"]= 'Click Me!'
```